deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,77 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
```

Comment:
Het is handig voor het overzicht om voor opdrachten met comments aan te geven voor welke opdracht de CSS is of voor grotere opdrachten aan te geven voor welke gedeelte van de pagina de CSS is

Reply:
Al moet ik daar ook bij zeggen dat de opdrachten in de class namen staan :DD